main.py

```diff
@@ -5,7 +5,14 @@
 
 # Function to allow backtracking to the menu after a cancelled "from path".
 def startMenu():
-	option = int(input("What do you want to do?\n1. List projects in current path\n2. Open path\n3. Exit\n"))
+	while True:
+		try:
+			option = int(input("What do you want to do?\n1. List projects in current path\n2. Open path\n3. Exit\n"))
+			break
+		except TypeError:
```

Comment:
Removing the TypeError part worked for me, otherwise anything but a number still breaks it

Reply:
Whoops meant `ValueError` not `TypeError`,
Although just doing a simple except statement is not a good idea as it will catch `KeyboardInterrupts`.